backend/test/test_crawlconfigs.py

```diff
@@ -173,6 +173,17 @@ def test_update_config_invalid_exclude_regex(
     assert r.json()["detail"] == "invalid_regex"
 
 
+def test_verify_default_select_links(
+    crawler_auth_headers, default_org_id, sample_crawl_data
+):
+    r = requests.get(
+        f"{API_PREFIX}/orgs/{default_org_id}/crawlconfigs/{cid}/",
+        headers=crawler_auth_headers,
+    )
+    assert r.status_code == 200
+    assert r.json()["config"]["selectLinks"] == "a[href]->href"
```

Comment:
```suggestion
    assert r.json()["config"]["selectLinks"] == ["a[href]->href"]
```

Update test to reflect change